upload/upload.go

```diff
@@ -95,30 +121,30 @@ func (u ImageUploader) Load(recipeID int64, imageName string) ([]byte, error) {
 	return io.ReadAll(file)
 }
 
-func (u ImageUploader) generateThumbnail(original image.Image, contentType string, saveDir string, imageName string) (string, error) {
-	thumbImage := imaging.Thumbnail(original, u.imgCfg.ThumbnailSize, u.imgCfg.ThumbnailSize, toResampleFilter(u.imgCfg.ThumbnailQuality))
-
+func (u ImageUploader) generateThumbnail(original image.Image, saveDir string, imageName string) (string, error) {
+	thumbImage := scaleImage(original, u.imgCfg.ThumbnailSize, u.imgCfg.ThumbnailQuality)
```

Comment:
This doesn't do quite what the old code did. This will preserve aspect ration whereas the old code clipped to the square box

Reply:
Can potentially use the SubImage() method to get the cropped image first